src/Components/SmartComponents/Reports/ReportsPage.js

```diff
@@ -92,6 +95,16 @@ const ReportsPage = () => {
         <FileAltIcon color="var(--pf-v5-global--link--Color)" />
     </Icon>;
 
+    const fetchOsStuff = async () => {
+        try {
+            const data = await getOSExposure();
+            setOsExposureData(data.data);
+            setOsExposureModalOpen(true);
+        } catch (error) {
+            return [];
+        }
+    };
```

Comment:
nit-pick: I would suggest to move this logic to the LifecycleSelect component instead and fetch OS exposure from there on mount using useEffect or other custom hook. Otherwise, it adds more complexity to the ReportsPage component which should only manage the available cards and modals open/close logic in my opinion.

Reply:
I'm assuming you mean move it into OsExposureReportModal?